### PR TITLE
DOCS-3005: Review Set capture status

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on Stoplight, Redocly
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: openapi/paths/orders@{order_id}.yaml [hunk 1]
```diff
@@ -122,25 +68,24 @@ patch:
...
-                type: boolean
+                $ref: '../components/schemas/Success.yaml'
               data: 
-                description: No data returned.
-                type: string                
-            type: object
-      description: ''
+                description: Empty
+                type: object
+                example: []
```
Comment (Sarah-MultiSafepay) [inline openapi/paths/orders@{order_id}.yaml:82]: We shouldn't add empty arrays. It upsets JSON parsers. Can we replace this with empty single quotes?

**Review by Sarah-MultiSafepay — COMMENTED** [review]
@FrancescaNCook, just one thing to change!

File: openapi/paths/orders@{order_id}.yaml (hunk 1 above)
Comment (FrancescaNCook) [inline openapi/paths/orders@{order_id}.yaml:82]:
```suggestion
                example: ''              
```

Comment (FrancescaNCook) [reply openapi/paths/orders@{order_id}.yaml:82]: @Sarah-MultiSafepay Good to know! Thank you!

**Pull request merged**